flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/cache/AbstractDownloadCache.java

```diff
@@ -102,13 +126,16 @@ public Path getOrDownload(final String url, final Path targetDir) throws IOExcep
 		} else {
 			final Path scopedDownloadDir = downloadsDir.resolve(String.valueOf(url.hashCode()));
 			Files.createDirectories(scopedDownloadDir);
-			log.info("Downloading {}.", url);
+
+			long downloadTimeoutMillis = getDownloadTimeoutMillis();
+			int maxRetries = getDownloadMaxRetries();
+			log.info("Downloading {} with maxRetries={}, and timeout of one retry is {} (ms).", url, maxRetries, downloadTimeoutMillis);
 			AutoClosableProcess
-				.create(
-					CommandLineWrapper.wget(url)
-						.targetDir(scopedDownloadDir)
-						.build())
-				.runBlocking(Duration.ofMinutes(2));
+					.create(
```

Comment:
revert indentation change

Reply:
The new indentation should be right here, so I think we can keep the change?

Reply:
we don't have a standard indentation; to prevent people from changing things back and forth we hence discourage indentation changes.